features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/SearchFragment.kt

```diff
@@ -175,6 +177,37 @@ class SearchFragment : Fragment(R.layout.fragment_search) {
         viewModel.allMerchantLocations.observe(viewLifecycleOwner) { merchantLocations ->
             merchantLocationsAdapter.submitList(merchantLocations)
         }
+        viewModel.syncProgressLiveDataFromFlow.observe(viewLifecycleOwner) { syncProgress ->
+
+            when (syncProgress.status) {
+                Status.LOADING -> {
+                    binding.apply {
+                        syncStatus.isVisible = true
+                        progress.isVisible = true
+                        progress.progress = syncProgress.data?.toInt() ?: 0
```

Comment:
The progress bar is indeterminate, which means this progress value is probably not used for anything.